app/models/concerns/vacancy_job_specification_validations.rb

```diff
@@ -2,12 +2,36 @@ module VacancyJobSpecificationValidations
   extend ActiveSupport::Concern
   include ApplicationHelper
 
+  MAX_INTEGER = 2147483647
```

Comment:
Does this need a freeze to make sure it's not accidentally mutated?

Reply:
@tahb no. Rubocop ` Style/RedundantFreeze: Do not freeze immutable objects, as freezing them has no effect.`